06-kotlin-mqtt/src/main/kotlin/ProtelisConfigSpec.kt

```diff
@@ -0,0 +1,13 @@
+package demo
+
+import com.uchuhimo.konf.ConfigSpec
+
+data class ProtelisNode(val id: Int, val listen: String, val neighbors: Set<String>, val leader: Boolean = false)
+
+object ProtelisConfigSpec : ConfigSpec("protelis") {
+    val iterations by required<Int>(description = "Number of iterations the nodes have to run.")
+    val protelisModuleName by required<String>()
+    val brokerHost by required<String>()
+    val brokerPort by required<Int>()
+    val nodes by required<List<ProtelisNode>>(description = "List of nodes")
+}
```

Comment:
There is no Kdoc in this file. Can you provide it for the public members?

Reply:
This is used by Konf to map the TOML object to a class, should I document it?

Reply:
Just a note on top, with the explanation you just gave me